src/main/java/kr/allcll/seatfinder/pin/PinService.java

```diff
@@ -0,0 +1,29 @@
+package kr.allcll.seatfinder.pin;
+
+import java.util.List;
+import kr.allcll.seatfinder.subject.Subject;
+import kr.allcll.seatfinder.subject.SubjectRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class PinService {
+
+    private final PinRepository pinRepository;
+    private final SubjectRepository subjectRepository;
+
+    public void addPinAtSubject(Long subjectId, String userToken) {
+        List<Pin> userPins = pinRepository.findAllByTokenId(userToken);
+        validatePinCount(userPins);
```

Comment:
이렇게 가면 어떨까요? 확신이 안 서서, 수민님의 의견이 궁금해요! 
```suggestion
validatePinsCount(userPins);
```